src/commands/configuration/togglepii.ts

```diff
@@ -1,14 +1,29 @@
 import { DB } from '@root/config';
 import { SageUser } from '@lib/types/SageUser';
-import { Message } from 'discord.js';
+import { CommandInteraction, Message } from 'discord.js';
 import { DatabaseError } from '@lib/types/errors';
 import { Command } from '@lib/types/Command';
 
 export default class extends Command {
 
-	description = `Toggles whether or not your personally identifiable information will be sent by instructors over Discord.`;
+	description = `Toggles whether your personally identifiable information will be sent to instructors over Discord.`;
```

Comment:
Clarifying what counts as personal information in extendedHelp, or even the description is the character limit allows, would be a good idea.